src/style-library/styles/_mixins.scss

```
@@ -63,3 +63,41 @@
 	text-transform: uppercase;
 	color: var(--kb-sl-color-text-primary);
 }
+
+// The frame half of a block-preset row preview that stands in for a colored surface (the Row Layout
+// band, the Section slab). Draws the border the preset's color is applied to, and a checker BEHIND
+// where the preset's background will be painted -- both blocks ship a transparent background, and
+// against the list's white surface a transparent background and a white one are indistinguishable, so
+// without the checker every unstyled preset would look like it had set white.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)